PSOperations/LocationCapability-iOS.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  LocationCapability.swift
+//  PSOperations
+//
+//  Created by Dev Team on 10/4/15.
+//  Copyright © 2015 Pluralsight. All rights reserved.
+//
+
+#if os(iOS) || os(watchOS)
+
+import Foundation
+import CoreLocation
+
+public enum Location: CapabilityType {
+    public static let name = "Location"
+    
+    case WhenInUse
+    case Always
```

Comment:
This is clever. I was wondering how you were going to solve the issue of different permissions levels. 👏